### ISSUE TITLE
Mumble 1.3 cannot connect.

### ISSUE THREAD

**Issue by FryDay** [umurmur/umurmur#82]
The latest version of Mumble built from the dev branch cannot connect to umurmur. The Older Encryption error is thrown. I can connect fine from Mumble 1.2.

**Comment by FryDay** [comment]
This is the server-side error: SSL handshake failed: -29312

**Comment by corrado113** [comment]
Since mumble 1.3.0 970 g888a459 snapshot I can no longer connect to my umurmur server (polarssl)

**Comment by snowblind** [comment]
Same here..
Getting this in log file: SSL handshake failed: -80

I am rolling back to previous snapshot version as last time this happen the next version of the snapshot released worked.

**Comment by fmorgner** [comment]
They require newer suites. Fixed it in my working copy, will push tomorrow.

**Comment by snowblind** [comment]
OK, I will try it when you do

**Comment by zao** [comment]
@fmorgner Could you please outline what you changed, so I can restore functionality to my deployment until it arrives upstream?

**Comment by mkrautz** [comment]
@fmorgner Hold on. 1.3.x shouldn't break backwards compatibility. Before you change anything, can you explain what's going on? I'd rather we fix it in the the snapshots...

**Comment by mkrautz** [comment]
@FryDay @corrado113: If you're building Mumble yourself, and you're building it against Qt 5.4 or earlier, it'll only do TLS 1.0.

Maybe that's the issue?

**Comment by corrado113** [comment]
I'm using the latest Windos x64 development snapshot from mumble.info, I'm not building it myself

**Comment by fmorgner** [comment]
uMurmur currently supports a lot of legacy suites which we shouldn't support anyways. Plus i figured that the OpenSSL backend is somewhat broken in that it does not support ECDHE suites due to no ECDHE key bein allocated. I fixed that plus removed a lot of suites. No worries though, i tried it with the 1.2 and 1.3 series and it seems to work fine now. Pushing any moment.

**Comment by fmorgner** [comment]
I've pushed a patch. @zao I basically upgraded the suites. Take a look at the diff in https://github.com/umurmur/umurmur/commit/c72ebd29f239793218f2248c3f8b48a7b3e0a541

**Comment by zao** [comment]
@fmorgner I'm on some form of raspbian on the Pi2, and apt claims I've got PolarSSL `1.2.9-1~deb7u6`.
Mumble `1.3.0-970~g888a459~snapshot` refuses to connect to umurmurd 779660334a7291f4918e313a78ed48f4d3d6e0a5.

**Comment by snowblind** [comment]
I run my mumble server on a rpi B1 running a older version of raspbian I stopped upgrading when the kernel got changed to to 4.0+. I don't feel like dealing with the problems it causes with the using the ports IO header yet.

I built and installed mbedTLS 2.2.1 and now can use newer mumble client snapshots.

I had to add --with-ssl=mbedtls to my cmake command..

@fmorgner  This issue seems to be closed now as supporting PolarSSL needs to be droped in any release going forward I think.
